apps/client/app/modules/photo/index.tsx

```diff
@@ -185,7 +196,14 @@ export const PhotoModule = () => {
 					</div>
 
 					<div className="mt-5">
-						<h1 className="font-bold">{content.title}</h1>
+						<div className="flex flex-row">
+							<h1 className="font-bold">{content.title}</h1> 
+							<PencilIcon className="ml-4 h-6 w-4 text-black-700" onClick={toggleModal}/>
+								{
+									openModal && (<EditTitleModal isOpened toggleModal={toggleModal}/>)
+								}
+						</div>
+						
```

Comment:
And then you can do this here:

```suggestion
							<PencilIcon className="ml-4 h-6 w-4 text-black-700" onClick={editTitleDialogState.onToggle}/>
								<EditTitleModal isOpened={editTitleDialogState.isOpened} toggleModal={editTitleDialogState.onToggle}/>
						</div>
						
```